### PR TITLE
chore(*): fix jekyll warning

### PR DESCRIPTION
**Pull request by rav3**
and also convert toast to alert component for `installation.html` & `customise-with-sass.html`

### CODE REVIEW

File: collections/_get-started/customise-with-sass.html [hunk 1]
```diff
@@ -52,10 +52,10 @@
 
 {% capture sgds-variable %}
 node_modules/
-  └── @govtechsg/sgds/sass/
-      └── bootstrap/
-      └── sgds-theme/
-          └── _sgds-variables.scss
+└── @govtechsg/sgds/sass/
```
Comment (clukhei) [inline collections/_get-started/customise-with-sass.html:55]: I thought the current folder structure (above in red) made more sense. how come bootstrap, sgds-theme folder and sgds-variable.scss file is at same level as @govtechsg/sgds

File: collections/_get-started/installation.html [hunk 2]
```diff
@@ -3,65 +3,74 @@
...
+
 {% capture reactjs %}
 npm install @govtechsg/sgds-react
 
-//not required if using cdn
+// not required if using cdn
 npm install @govtechsg/sgds-govtech bootstrap-icons
 {% endcapture %}
-{% capture using-node %}
-npm i @govtechsg/sgds bootstrap-icons
```
Comment (clukhei) [inline collections/_get-started/installation.html:13]: should be npm install **@govtechsg/sgds** bootstrap-icons 

not, npm install @govtechsg/sgds-govtech bootstrap icons

**Review by clukhei — CHANGES_REQUESTED** [review]

File: collections/_get-started/customise-with-sass.html (hunk 1 above)
Comment (rav3) [reply collections/_get-started/customise-with-sass.html:55]: Think i formatted it in vscode. Will revert

File: collections/_get-started/installation.html (hunk 2 above)
Comment (rav3) [reply collections/_get-started/installation.html:13]: resolved in commit [c4e7d4f](https://github.com/GovTechSG/sgds/pull/372/commits/c4e7d4f332ef6c0e50a7429b0a143a05ad8981ba)

File: collections/_get-started/customise-with-sass.html (hunk 1 above)
Comment (rav3) [reply collections/_get-started/customise-with-sass.html]: resolved in commit [c84b9e6](https://github.com/GovTechSG/sgds/pull/372/commits/c84b9e6cc8c0cf24e86023430ae0b869aee2ea32)

**Pull request merged**